### ISSUE TITLE
Throws error when table name has - (dash) in it

### ISSUE THREAD

**Issue by pntrivedy** [barrymichaeldoyle/supabase-react-query-codegen#7]
1. Update table name with - (dash) in it. 
2. Run 'npx supabase-react-query-codegen generate supabase-react-query-codegen.config.json'

How to quickly fix this?

```
SyntaxError: '=' expected. (9:22)
   7 | export type UpdateFinanceRequest = { id: string; changes: { account_id?: number | null | undefined; amount?: number | null | undefined; category?: string | null | undefined; created_at?: string | undefined; due_at?: string | null | undefined; flow?: string | null | undefined; frequency?: string | null | undefined; id?: number | undefined; leverage?: string | null | undefined; links?: import("/Users/prerak/Sites/localhost/growth/src/types/types").Json | undefined; loanInt?: number | null | undefined; title?: string | null | undefined; type?: string | null | undefined; updated_at?: string | null | undefined; } };
   8 |     
>  9 | export type Financial-account = { amount: number | null; created_at: string; id: number; links: import("/Users/prerak/Sites/localhost/growth/src/types/types").Json; name: string | null; number: string
```

Thank you

**Comment by barrymichaeldoyle** [comment]
Thanks so much for bringing this up and making a PR :) I'll bump this version and deploy this soon!

**Comment by barrymichaeldoyle** [comment]
Now available in `v1.1.0`
